components/routine-logs-summery-view.js

```diff
@@ -1,12 +1,33 @@
 import { Grid } from "@mui/material";
+import { useEffect, useState } from "react";
 import KeyValueView from "./key-value-view";
+import { useRecoilValue } from 'recoil';
+import { selectedRoutineState } from "../utils/shared_atoms";
+
+export default function RoutineLogsSummeryView() {
+    const selectedRoutine = useRecoilValue(selectedRoutineState);
+    const [logsCounter, setLogsCounter] = useState({
+      exceptions: 0,
+      warnings: 0, 
+      info: 0, 
+      avg_fps: 0
+  })
+
+    useEffect(() => {
+      if (selectedRoutine !== null) {
```

Comment:
I would rather have this condition because it would be an error from the request, it would be an error from the parameters of the request.
and in a case where we deselect a routine and the variable is Null we don't want to send an api call.